packages/firebase_messaging/firebase_messaging_platform_interface/test/notification_test.dart

```diff
@@ -6,37 +6,226 @@ import 'package:firebase_messaging_platform_interface/firebase_messaging_platfor
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  group('Notification', () {
-    test('new instance', () {
-      const testTitle = 'test notification';
-      const notification = RemoteNotification(title: testTitle);
-      expect(notification.title, equals(testTitle));
+  Map<String, dynamic>? mockNotificationMap;
+  Map<String, dynamic>? mockNullNotificationMap;
```

Comment:
I don't like the idea of sharing those variables across all tests, as they'll likely change over time.

I'd declare those maps directly inside the tests that needs them